projects/fsastorefrontstyles/scss/components/checkout/_mini-cart.scss

```diff
@@ -31,3 +31,37 @@
     }
   }
 }
+
+.action-wrapper {
+  display: flex;
+  justify-content: space-between;
+  cx-fs-cart-coupon {
+    width: 60%;
+  }
+  cx-fs-cms-sync-pilot {
+    width: max-content;
+    padding-inline-end: 1rem;
+    h5 {
+      font-size: 0.875rem;
```

Comment:
Did you perhaps mean 0.857rem which should be used as `var(--cx-text-font-size)`? And also check if it then needs to be a `h5` tag. It can be a regular `span` or `p` tag?

Reply:
I didn't see we have a defined variable for that font size. I have chosen `h5` since it is the title of the component and also I need font-weight rule. I can use `span` and add some additional rules.

Reply:
Although it can be span with `h5` or `semi-bold` class, which would give it the right font-weight, if we look at it as a separate component then you are right to use tag for this. Just then fix the size with variable.